ProcessLib/Parameter/Parameter.h

```diff
@@ -68,6 +75,40 @@ struct Parameter : public ParameterBase
     //! Returns the parameter value at the given time and position.
     virtual std::vector<T> const& operator()(
         double const t, SpatialPosition const& pos) const = 0;
+
+    //! Returns a matrix of values for all nodes of the given element.
+    //
+    // The matrix is of the shape CxN, where C is the number of components and N
+    // the number of nodes, such that subsequent multiplication with shape
+    // functions matrix results in a vector of length C.
```

Comment:
In many occasions the multiplication of shape (row) vectors `N` and nodal values (column vectors) `V` is currently done as `N * V`. Due to consistency reasons therefore it might be better to transpose the matrix order in this method. Furthermore, often we use row major storage.

Reply:
Changed the result to be NxC, same as *e.g.* the local_x scalar quantities T or p.

Reply:
OK, thanks.